### DISCUSSION TITLE
讨论翻译的238页中的e=255

### DISCUSSION THREAD

**Discussion by liulikui** [kanition/pbrtbook#175] [交流]
238页中，应该是eb=255吧，原文没有错

**Comment by kanition** [comment]
前文已经说明eb = e - 127，而在e=255下浮点数具有特殊定义：若有效数字全为零，则表示无穷大；否则表示NaN。

根据原文上下文推断，文中应该继续介绍e=255下的后一种表示NaN的情况。此时eb=128，而不是误写的eb=255。不过这种情况eb已经没有用处了，因为不管无穷大还是NaN都已经不关心指数的具体大小了。

此外，根据wiki查询的资料，IEEE 754也确实这样规定e=255的含义的。

所以我对原文作出了这样的笔误修改。

**Comment by kanition** [comment]
说句题外话，我注意到你报错的位置和我现在最新的版本有一些页数差别，如果只是想阅读本书，建议直接下载最新版PDF。
